xt/lib/Pherkin/Extension/LedgerSMB.pm

```diff
@@ -202,25 +213,25 @@ sub create_template {
     $self->template_created(1);
 }
 
-
 sub ensure_template {
     my ($self) = @_;
 
     $self->create_template
         unless $self->template_created;
 }
 
-
 sub create_from_template {
     my ($self, $company) = @_;
 
-    my $template = $self->template_db_name;
-    $self->ensure_nonexisting_company($company);
-    $self->super_dbh->do(qq(CREATE DATABASE "$company" TEMPLATE "$template"));
-
     $self->last_feature_stash->{"the company"} = $company;
```

Comment:
Doesn't this break the callers contact? The result of this call is that the company does not exist after the call. I would say that there should be a modification to the callers to determine this call should not be made. What is the reason  you did it this way.